### PR TITLE
Change pycommon to use swsscommon

### PR DESCRIPTION
**Pull request by liuh-80**
#### Why I did it
    Switch py-common from swsssdk to swsscommon.

#### How I did it
    Change code and make file to use swsscommon.

#### How to verify it
    Pass all UT and E2E test.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111

### CODE REVIEW

**Comment by liuh-80** [comment]
py_common change will break UT in sonic-platform-daemons, the issue will be fix in this PR: https://github.com/Azure/sonic-platform-daemons/pull/246

**Comment by liuh-80** [comment]
This PR is continue of original PR: https://github.com/Azure/sonic-buildimage/pull/7655
And because this PR require many submodule change, for example sonic-utilities change, so this PR will keep in draft status until all change in submodule been merge to master branch, and pass all validation.

**Comment by liuh-80** [comment]
> > import glob
> 
> I notice the filename not proper
> 
> https://github.com/Azure/sonic-buildimage/blob/8d419ca2c59b0f75c24ba4ed73fbdc7df7812260/src/sonic-py-common/tests/device_info_test.py#L16
> 
> .
> Change mock_swsssdk to mock_swsscommon
> 
> Refers to: src/sonic-py-common/sonic_py_common/device_info.py:1 in [1a3a020](https://github.com/Azure/sonic-buildimage/commit/1a3a020c163ed109ec7b2276a93c95b569e14e8a). [](commit_id = [1a3a020](https://github.com/Azure/sonic-buildimage/commit/1a3a020c163ed109ec7b2276a93c95b569e14e8a), deletion_comment = False)

Fixed.

**Review by xumia — APPROVED** [review]

**Comment by liuh-80** [comment]
UT failed because check multi asic API will initialize ConfigDB config, need fix in sonic-snmpagent: https://github.com/Azure/sonic-snmpagent/pull/254

**Comment by liuh-80** [comment]
After merge with latest master branch code, only 1 UT failed and seems not related with code change in this PR: bgp/test_bgp_update_timer.py

**Comment by liuh-80** [comment]
After re-run, still following UT failed:
2022-05-19T00:58:10.3522829Z ERROR bgp/test_bgp_fact.py::test_bgp_facts[128-vlab-02-None] - Failed: Proces...

And same UT failed happened on many other PRs, seems a common issue, will re-run later.

File: rules/sonic-py-common.mk [hunk 1]
```diff
@@ -4,6 +4,8 @@ ifeq ($(ENABLE_PY2_MODULES), y)
     SONIC_PY_COMMON_PY2 = sonic_py_common-1.0-py2-none-any.whl
     $(SONIC_PY_COMMON_PY2)_SRC_PATH = $(SRC_PATH)/sonic-py-common
     $(SONIC_PY_COMMON_PY2)_DEPENDS += $(SWSSSDK_PY2)
+    $(SONIC_PY_COMMON_PY2)_DEBS_DEPENDS = $(LIBSWSSCOMMON) \
```
Comment (qiluo-msft) [inline rules/sonic-py-common.mk:7]: >LIBSWSSCOMMON [](http://example.com/codeflow?start=44&length=13)

Does it depend on LIBSWSSCOMMON directly?

Comment (liuh-80) [reply rules/sonic-py-common.mk:7]: Fixed, remove indirect dependency to LIBSWSSCOMMON.

**Review by qiluo-msft — APPROVED** [review]

**Comment by dgsudharsan** [comment]
Request for 202205

**Comment by qiluo-msft** [comment]
@vivekrnv @dgsudharsan Backport of this feature include many PRs in multiple repos, and with dependencies between them. If there is no strong motivation, we will not backport to 202205 and earlier branches.

**Pull request merged**